docs/getting-started/quickstart-existing-project.md

```diff
@@ -203,7 +203,7 @@ Next, add the following contents to your `tsconfig.json`:
   "compilerOptions": {
     "outDir": "dist",
     "rootDir": "src",
-    "lib": ["esnext"],
+    "lib": ["esnext", "dom"],
```

Comment:
I would rather add `@types/node` to the project dependencies than adding `dom` to the tsconfig as we are running Prisma inside node context and not inside of a dom context.

Can you please instead modify step 3 and add `@types/node` where we have `pm install typescript ts-node --save-dev` 🙏